build.gradle

```diff
@@ -244,6 +244,8 @@ subprojects.forEach {
 			.dependsOn(processResources)
 	// make them always run, because if they don't, extreme weirdness with caching occurs with every other task
 			.outputs.upToDateWhen { false }
+
+	it.tasks.findByName("sourcesJar").duplicatesStrategy = DuplicatesStrategy.WARN
```

Comment:
only add this to the subproject it fails in. I have the build and workflow fixed locally and will push shortly, please merge it and remove this.